src/message.rs

```diff
@@ -28,15 +31,129 @@ fn _message_prettify(message: CString, comment_char: Option<u8>) -> Result<Strin
     Ok(ret.as_str().unwrap().to_string())
 }
 
+/// Collection of trailer key–value pairs.
+///
+/// Use `iter()` to get access to the values.
+pub struct MessageTrailers<'pair> {
```

Comment:
I believe that this is a uniquely owned value, so I don't think that a `'pair` lifetime parameter here is necessary.